compiler/one-cmds/tests/onecc_043.test

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test for "O1=True" option in onecc config file
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+configfile="onecc_043.cfg"
+outputfile="inception_v3.onecc_043.opt.circle"
+
+rm -f ${filename}.log
+rm -rf ${outputfile}
+
+# run test
+onecc -C ${configfile} > ${filename}.log 2>&1
+
+if [[ ! -s "${outputfile}" ]]; then
+  trap_err_onexit
+fi
+
```

Comment:
Maybe we need to check the O1 option. #10472 can apply this codes as well. 

```suggestion
declare -a O1_OPTS=(
  # Constant folding
  "fold_add_v2" "fold_cast"
  "fold_densify" "fold_dequantize"
  "fold_dwconv" "fold_fully_connected"
  "fold_gather" "fold_sparse_to_dense"

  # Operator fusion
  "fuse_add_with_tconv" "fuse_add_with_fully_connected"
  "fuse_batchnorm_with_conv" "fuse_batchnorm_with_dwconv"
  "fuse_batchnorm_with_tconv" "fuse_activation_function"
  "fuse_instnorm" "fuse_prelu"
  "fuse_mean_with_mean" "fuse_transpose_with_mean"
  "transform_min_max_to_relu6" "transform_min_relu_to_relu6"

  # Remove redundant operators
  "remove_redundant_reshape" "remove_redundant_transpose"
  "remove_unnecessary_reshape" "remove_unnecessary_slice"
  "remove_unnecessary_strided_slice" "remove_unnecessary_split"

  # Canonicalization
  "resolve_customop_add" "resolve_customop_batchmatmul"
  "resolve_customop_matmul" "resolve_customop_max_pool_with_argmax"
  "resolve_customop_splitv" "substitute_pack_to_reshape"
  "substitute_padv2_to_pad" "substitute_splitv_to_split"
  "substitute_squeeze_to_reshape" "substitute_strided_slice_to_reshape"
  "substitute_transpose_to_reshape" "forward_reshape_to_unaryop"
  "forward_transpose_op" "replace_non_const_fc_with_batch_matmul"
)

declare -a NO_O1_OPTS=(
  "convert_nchw_to_nhwc" "expand_broadcast_const"
  "nchw_to_nhwc_input_shape" "nchw_to_nhwc_output_shape"
  "fuse_bcq" "fuse_preactivation_batchnorm"
  "make_batchnorm_gamma_positive" "replace_cw_mul_add_with_depthwise_conv"
  "remove_fakequant" "remove_quantdequant"
  "remove_redundant_quantize" "replace_sub_with_add"
  "shuffle_weight_to_16x1float32" "unroll_unidirseqlstm"
)


for opt in "${O1_OPTS[@]}"
do
  if ! grep -q ${opt} ${outputfile}.log; then
    trap_err_onexit
  fi
done

for no_opt in "${NO_O1_OPTS[@]}"
do
  if grep -q ${no_opt} ${outputfile}.log; then
    trap_err_onexit
  fi
done
```

Reply:
> Maybe we need to check the O1 option.

Can you explain more about this?

Reply:
Hmm.. this code has to be updated if the list of options in `O1` is changed.

Reply:
> > Maybe we need to check the O1 option.
> 
> Can you explain more about this?

This test will be passed even though O1 option turns off. So, I think we need to check if the option turns off well.

Reply:
> Hmm.. this code has to be updated whenever the list of options in `O1` are changed, which is too burdensome.
> 
> We may find another way (make a single file that maintains `O1` list) to implement this check.

I thought so at first. Hmm.. but checking it seems necessary.

Reply:
> I thought so at first. Hmm.. but checking it seems necessary.

I agree with this. I'll make another PR for that.

Reply:
> So, I think we need to check if the option turns off well.

If this test is for what @mhs4670go explained, I think it would be better to check in some unit(?) test not in this end-user one-cmds test.

Reply:
@seanshpark I guess that what @mhs4670go wanted to say is, we have to check the individual options of `O1` are turned on (except `fold_add_v2=False` in the cfg file).

And if we hard-code the list of optimization options in each test file, we will have to update the test files whenever `O1` is updated. To address this, I made https://github.com/Samsung/ONE/pull/10474, which makes a single source of optimization list for `O1` (= `O1.txt`). The file will be copied to test directory and used by this test :)

I'll rebase this PR after #10474 is landed.